internal/config/types.go

```diff
@@ -54,6 +54,8 @@ type ServiceInfo struct {
 	EnableAsyncReadings bool
 	// AsyncBufferSize defines the size of asynchronous channel
 	AsyncBufferSize int
+	// MaxRequestSize defines the maximum size of http request body in bytes
+	MaxRequestSize int64
```

Comment:
Since this only applies to Device command, should the name be more specific. i.e. one would assume this apply to all routes by its general name...

Reply:
Related to below comment: if we agree to apply this only on Device command, maybe rename to `MaxDeviceCommandRequestSize` ?

Reply:
this needs to be synced with @iain-anderson

Reply:
I think the purpose of it is to defend against denial of service with oversize requests - so it should apply to all routes